### PR TITLE
Add parsing of 1.14.4

### PR DESCRIPTION
**Pull request by killerwife**
Update fields will not work and I am not going near that

### CODE REVIEW

File: WowPacketParser/Misc/ClientVersion.cs [hunk 1]
```diff
@@ -1084,6 +1084,8 @@ public static ClientVersionBuild GetVersionDefiningBuild(ClientVersionBuild buil
                 case ClientVersionBuild.V1_14_3_48611:
                 case ClientVersionBuild.V1_14_3_49229:
                 case ClientVersionBuild.V1_14_3_49821:
+                case ClientVersionBuild.V1_14_4_51146:
+                case ClientVersionBuild.V1_14_4_51535:
```
Comment (ratkosrb) [inline WowPacketParser/Misc/ClientVersion.cs:1088]: Isn't 1.14.4 rebased upon Dragonflight? It ought to be grouped with 3.4 not with 2.5, or it won't inherit packet handlers from Dragonflight.

Comment (funjoker) [reply WowPacketParser/Misc/ClientVersion.cs]: well 3.4.0 was SL
3.4.3 is DF.

It's a mess tbh dont mess with all the inherit packet handlers :S

Comment (killerwife) [reply WowPacketParser/Misc/ClientVersion.cs]: I am doing it properly in last commit

**Comment by killerwife** [comment]
Would really appreciate if someone could look at it and improve it or merge it. Since seems like only I and my team are using latest vanilla parsing actively.

**Comment by Daribon** [comment]
The SMSG_ON_MONSTER_MOVE fix should be applied for all 1.15x builds. Already in 1.15.0 PTR build 52212 SMSG_ON_MONSTER_MOVE will not parse without the fix.

**Comment by killerwife** [comment]
Rebased against cata changes

**Pull request merged**